src/main/java/jsr223/docker/compose/DockerComposeCommandCreator.java

```diff
@@ -1,47 +1,41 @@
 package jsr223.docker.compose;
 
+import jsr223.docker.compose.utils.DockerComposePropertyLoader;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
 import java.util.ArrayList;
 import java.util.List;
 
-import jsr223.docker.compose.utils.DockerComposePropertyLoader;
-
+@NoArgsConstructor
 public class DockerComposeCommandCreator {
 
     // Constants
-    @lombok.Getter
-    private static String yamlFileName = "docker-compose.yml";
-    private static String filenameArgument = "-f";
-    private static String stopContainerArgument = "stop";
-    private static String removeContainerArgument = "rm";
-    private static String removeWithForceContainerArgument = "--force";
-    private static String setupContainerArgument = "up";
-
-
-    /**
-     * Construct the docker compose stop command.
-     *
-     * @return String array representing a command.
-     */
-    public static String[] createDockerComposeStopCommand() {
-        List<String> command = new ArrayList<>();
-        addSudoAndDockerComposeCommand(command);
+    @Getter
+    private final String yamlFileName = "docker-compose.yml";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1170)